footer.php

```diff
@@ -14,7 +14,7 @@
 
 	<?php do_action( 'storefront_before_footer' ); ?>
 
-	<footer id="colophon" class="site-footer" role="contentinfo">
```

Comment:
I'm not sure anymore about removing role attributes on the specific elements elements such as `<header>`, `<footer>`, `<nav>`, etc... is a good thing.
From https://www.w3.org/WAI/GL/wiki/Using_HTML5_nav_element: 

> However, in order to make the structure more accessible to user agents that support ARIA as well as ensuring that user agents that don't support HTML5 can also understand the structure, adding the ARIA role="navigation" is advised.

Which just contradict their own tool "Nu HTML Checker" throwing warnings saying:

> Element nav does not need a role attribute.

...